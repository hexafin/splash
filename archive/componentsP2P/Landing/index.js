import Landing from "./Landing"
import {connect} from "react-redux"
import {bindActionCreators} from "redux"
import {ClaimUsername, InviteFriends, getDeepLinkedSplashtag, SmsAuthenticate} from "../../actions/waitlist"

const mapStateToProps = (state) => {
    return {
      waitlisted: state.general.waitlisted,
      splashtagOnHold: state.waitlist.splashtagOnHold,
      phoneNumber: state.waitlist.phoneNumber,
      username: state.general.person.username,
      smsAuthenticated: state.waitlist.smsAuthenticated
    }
}

const mapDispatchToProps = (dispatch) => {
    return bindActionCreators({
        ClaimUsername, InviteFriends, getDeepLinkedSplashtag, SmsAuthenticate
    }, dispatch)
}

export default connect(mapStateToProps, mapDispatchToProps)(Landing)
