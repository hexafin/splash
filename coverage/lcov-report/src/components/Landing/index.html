<!doctype html>
<html lang="en">
<head>
    <title>Code coverage report for src/components/Landing</title>
    <meta charset="utf-8" />
    <link rel="stylesheet" href="../../../prettify.css" />
    <link rel="stylesheet" href="../../../base.css" />
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <style type='text/css'>
        .coverage-summary .sorter {
            background-image: url(../../../sort-arrow-sprite.png);
        }
    </style>
</head>
<body>
<div class='wrapper'>
  <div class='pad1'>
    <h1>
      <a href="../../../index.html">All files</a> src/components/Landing
    </h1>
    <div class='clearfix'>
      <div class='fl pad1y space-right2'>
        <span class="strong">63.27% </span>
        <span class="quiet">Statements</span>
        <span class='fraction'>31/49</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">41.67% </span>
        <span class="quiet">Branches</span>
        <span class='fraction'>15/36</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">46.15% </span>
        <span class="quiet">Functions</span>
        <span class='fraction'>6/13</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">61.7% </span>
        <span class="quiet">Lines</span>
        <span class='fraction'>29/47</span>
      </div>
    </div>
    <p class="quiet">
      Press <em>n</em> or <em>j</em> to go to the next uncovered block, <em>b</em>, <em>p</em> or <em>k</em> for the previous block.
    </p>
  </div>
  <div class='status-line medium'></div>
<div class="pad1">
<table class="coverage-summary">
<thead>
<tr>
   <th data-col="file" data-fmt="html" data-html="true" class="file">File</th>
   <th data-col="pic" data-type="number" data-fmt="html" data-html="true" class="pic"></th>
   <th data-col="statements" data-type="number" data-fmt="pct" class="pct">Statements</th>
   <th data-col="statements_raw" data-type="number" data-fmt="html" class="abs"></th>
   <th data-col="branches" data-type="number" data-fmt="pct" class="pct">Branches</th>
   <th data-col="branches_raw" data-type="number" data-fmt="html" class="abs"></th>
   <th data-col="functions" data-type="number" data-fmt="pct" class="pct">Functions</th>
   <th data-col="functions_raw" data-type="number" data-fmt="html" class="abs"></th>
   <th data-col="lines" data-type="number" data-fmt="pct" class="pct">Lines</th>
   <th data-col="lines_raw" data-type="number" data-fmt="html" class="abs"></th>
</tr>
</thead>
<tbody><tr>
	<td class="file low" data-value="Landing.js"><a href="Landing.js.html">Landing.js</a></td>
	<td data-value="31.82" class="pic low"><div class="chart"><div class="cover-fill" style="width: 31%;"></div><div class="cover-empty" style="width:69%;"></div></div></td>
	<td data-value="31.82" class="pct low">31.82%</td>
	<td data-value="22" class="abs low">7/22</td>
	<td data-value="22.73" class="pct low">22.73%</td>
	<td data-value="22" class="abs low">5/22</td>
	<td data-value="50" class="pct medium">50%</td>
	<td data-value="8" class="abs medium">4/8</td>
	<td data-value="31.82" class="pct low">31.82%</td>
	<td data-value="22" class="abs low">7/22</td>
	</tr>

<tr>
	<td class="file high" data-value="LandingSwipeView.js"><a href="LandingSwipeView.js.html">LandingSwipeView.js</a></td>
	<td data-value="88.89" class="pic high"><div class="chart"><div class="cover-fill" style="width: 88%;"></div><div class="cover-empty" style="width:12%;"></div></div></td>
	<td data-value="88.89" class="pct high">88.89%</td>
	<td data-value="27" class="abs high">24/27</td>
	<td data-value="71.43" class="pct medium">71.43%</td>
	<td data-value="14" class="abs medium">10/14</td>
	<td data-value="40" class="pct low">40%</td>
	<td data-value="5" class="abs low">2/5</td>
	<td data-value="88" class="pct high">88%</td>
	<td data-value="25" class="abs high">22/25</td>
	</tr>

</tbody>
</table>
</div><div class='push'></div><!-- for sticky footer -->
</div><!-- /wrapper -->
<div class='footer quiet pad2 space-top1 center small'>
  Code coverage
  generated by <a href="https://istanbul.js.org/" target="_blank">istanbul</a> at Fri Apr 19 2019 16:14:10 GMT-0400 (Eastern Daylight Time)
</div>
</div>
<script src="../../../prettify.js"></script>
<script>
window.onload = function () {
        if (typeof prettyPrint === 'function') {
            prettyPrint();
        }
};
</script>
<script src="../../../sorter.js"></script>
<script src="../../../block-navigation.js"></script>
</body>
</html>
