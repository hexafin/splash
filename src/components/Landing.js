import React, {Component} from "react"
import {Actions} from "react-native-router-flux";
import {connect} from "react-redux";
import {
    View,
    Text,
    StyleSheet,
    Image
} from "react-native"
import Button from "./universal/Button"
import {LoadApp} from "../actions/general"
import {colors} from "../lib/colors"
import {defaults} from "../lib/styles"
import TimerMixin from 'react-timer-mixin'

class Landing extends Component {

    constructor(props) {
        super(props)

    }

    componentDidMount() {
        if (this.props.authenticated) {
            Actions.home()
        }
    }

    render() {
        return (
            <View style={styles.container} onPress={() => LoadApp()}>

                <Image style={styles.loadingLogo} source={require("../lib/images/splash-logo-xlarge.png")}/>

                <Button style={styles.landingButton} onPress={() => Actions.splash()} title="Sign Up"/>

                <Button style={styles.landingButton} onPress={() => Actions.home()} title="Login"/>

            </View>
        )
    }

}


const styles = StyleSheet.create({
    container: {
        ...defaults.container,
        flexDirection: "column",
        justifyContent: "space-around",
        alignItems: "center"
    },
    loadingLogo: {
        width: 400,
        height: 400
    },
    landingButton: {
        flex: 1
    }
})

const mapStateToProps = (state) => {
    return {
        authenticated: state.general.authenticated
    }
}

export default connect(mapStateToProps)(Landing)