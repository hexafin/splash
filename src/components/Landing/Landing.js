import React, {Component} from "react"
import {
    View,
    Text,
    StyleSheet,
    Image,
    TouchableOpacity
} from "react-native"
import {colors} from "../../lib/colors"
import {defaults, icons} from "../../lib/styles"
import { Linking, AppState } from 'react-native'


class Landing extends Component {
  componentDidMount() {
    AppState.addEventListener('change', this.handleAppStateChange);
    Linking.getInitialURL().then(url => console.log('initial', url));
  }

  componentWillUnmount() {
    AppState.removeEventListener('change', this.handleAppStateChange);
  }

  handleAppStateChange = (currentAppState) => {
    Linking.getInitialURL().then(url => this.handleDeepLink({ url }));
  };

  handleDeepLink = (event) => {
	  const parts = (event.url).split('/')
    console.log(parts);
  };
    render() {

        return (
            <View style={styles.container}>

                <Image style={styles.wavesImage}
                    source={require("../../assets/images/waves.png")}/>

                <View style={styles.header}>
                    <View style={styles.logoWrapper}>
                        <Image source={icons.splash}
                            style={styles.logo}/>
                        <Text style={styles.logoText}>Splash</Text>
                    </View>
                    <View style={styles.slogan}>
                        <Text style={styles.sloganText}>
                            Splash is your wallet
                        </Text>
                        <Text style={styles.sloganSubText}>
                            Make it personal with a splashtag
                        </Text>
                    </View>
                </View>

                <View style={styles.floating}></View>

                <View style={styles.footer}>
                    <TouchableOpacity style={styles.footerButton}
                        onPress={() => {this.props.navigation.navigate("ChooseSplashtag")}}>
                        <Text style={styles.footerButtonText}>
                            Claim your splashtag
                        </Text>
                    </TouchableOpacity>
                </View>


            </View>
        )

    }

}

const styles = StyleSheet.create({
    container: {
        ...defaults.container,
        justifyContent: "space-between",
        position: "relative"
    },
    wavesImage: {
        position: "absolute",
        bottom: -50,
        left: 0,
        right: 0,
        width: 400,
        height: 400
    },
    header: {
        flex: 1,
        padding: 30,
        flexDirection: "column",
    },
    logoWrapper: {
        flexDirection: "row",
        justifyContent: "center",
        alignItems: "center",
        padding: 30
    },
    logo: {
        height: 36,
        width: 28,
        marginRight: 10
    },
    logoText: {
        fontSize: 28,
        paddingBottom: 4,
        fontWeight: '400',
        color: colors.nearBlack
    },
    slogan: {
        flexDirection: "column",
        alignItems: "center",
        padding: 0,
        marginTop: 30
    },
    sloganText: {
        fontSize: 34,
        fontWeight: '500',
        color: colors.nearBlack,
        marginBottom: 20,
        textAlign: "center"
    },
    sloganSubText: {
        fontSize: 24,
        textAlign: "center",
        color: colors.lightGray
    },
    floating: {},
    footer: {
        padding: 20,
        marginBottom: 10
    },
    footerButton: {
        backgroundColor: colors.white,
        borderRadius: 5,
        flex: 1,
        padding: 30,
        justifyContent: "center",
        alignItems: "center"
    },
    footerButtonText: {
        fontSize: 24,
        color: colors.purple
    }
})

export default Landing
