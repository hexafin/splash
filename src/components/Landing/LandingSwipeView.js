import React, { Component } from 'react'
import {
    View,
    Text,
    StyleSheet,
    Dimensions,
    Image,
    Alert,
    Animated,
    TouchableOpacity,
    TouchableWithoutFeedback,
    processColor
} from 'react-native'
import { colors } from '../../lib/colors'
import { defaults, icons } from '../../lib/styles'
import { Input } from '../universal/Input'
import TouchID from 'react-native-touch-id'
import Button from '../universal/Button'
import LottieView from 'lottie-react-native'
import NextButton from '../universal/NextButton'
import { isIphoneX } from 'react-native-iphone-x-helper'
import LinearGradient from 'react-native-linear-gradient'
const image1 = require('../../assets/images/screen-view1.png')
const image2 = require('../../assets/images/screen-view2.png')
const image3 = require('../../assets/images/screen-view3.png')
const SCREEN_WIDTH = Dimensions.get('window').width
const SCREEN_HEIGHT = Dimensions.get('window').height
const xOffset = new Animated.Value(0)

class LandingSwipeView extends Component {
    constructor(props) {
        super(props)
    }

    render() {
        const progress = xOffset.interpolate({
            inputRange: [0, 750],
            outputRange: [0.2, 0.8]
        })

        let dots = []
        for (let i = 0; i < 3; i++) {
            const dotOpacity = xOffset.interpolate({
                inputRange: [
                    (i - 1) * SCREEN_WIDTH,
                    i * SCREEN_WIDTH,
                    (i + 1) * SCREEN_WIDTH
                ],
                outputRange: [0.25, 1, 0.25],
                extrapolate: 'clamp'
            })

            const thisDot = (
                <Animated.View
                    key={`${i}-dot`}
                    style={[
                        styles.dot,
                        {
                            opacity: dotOpacity
                        }
                    ]}
                />
            )
            dots.push(thisDot)
        }

        let images = []
        for (let i = 0; i < 3; i++) {
            const dotOpacity = xOffset.interpolate({
                inputRange: [
                    (i - 1) * SCREEN_WIDTH,
                    i * SCREEN_WIDTH,
                    (i + 1) * SCREEN_WIDTH
                ],
                outputRange: [0, 1, 0],
                extrapolate: 'clamp'
            })

            const thisImage = (
                <Animated.View
                    key={`${i}-dot`}
                    style={[
                        styles.image,
                        {
                            opacity: dotOpacity
                        }
                    ]}
                >
                    {i == 0 && (
                        <Image source={image1} style={styles.imageStyle} />
                    )}
                    {i == 1 && (
                        <Image source={image2} style={styles.imageStyle} />
                    )}
                    {i == 2 && (
                        <Image source={image3} style={styles.imageStyle} />
                    )}
                </Animated.View>
            )
            images.push(thisImage)
        }

        return (
            <Animated.View style={[styles.container]}>
                <View style={styles.imageContainer}>{images}</View>

                <View style={styles.dotContainer}>{dots}</View>

                <Animated.ScrollView
                    scrollEventThrottle={16}
                    showsHorizontalScrollIndicator={false}
                    onScroll={Animated.event(
                        [
                            {
                                nativeEvent: {
                                    contentOffset: { x: xOffset }
                                }
                            }
                        ],
                        { useNativeDriver: true }
                    )}
                    horizontal
                    pagingEnabled
                >
                    <Screen
                        text="Splash is your cryptocurrency wallet."
                        index={0}
                    />
                    <Screen
                        text={`Send money to anyone.\nBuy anything.`}
                        index={1}
                    />
                    <Screen
                        text={`Bye, bye big banks.\nHello Splash.`}
                        index={2}
                    />
                </Animated.ScrollView>

                <View style={styles.button}>
                    <Button
                        onPress={() =>
                            this.props.navigation.navigate('ChooseSplashtag')
                        }
                        round
                        large
                        title="Get your wallet"
                    />
                </View>
            </Animated.View>
        )
    }
}

const styles = StyleSheet.create({
    container: {
        flex: 1,
        backgroundColor: colors.primary
    },
    scrollView: {
        flexDirection: 'row',
        flex: 1
    },
    scrollPage: {
        width: SCREEN_WIDTH,
        padding: 20
    },
    dotContainer: {
        position: 'absolute',
        zIndex: 2,
        top: isIphoneX() ? SCREEN_HEIGHT * 0.2 : SCREEN_HEIGHT * 0.18,
        left: 50,
        flexDirection: 'row'
    },
    imageContainer: {
        position: 'absolute',
        zIndex: 0,
        bottom: 0,
        flex: 1,
        backgroundColor: 'yellow'
    },
    image: {
        position: 'absolute',
        bottom: 0
    },
    imageStyle: {
        height: SCREEN_HEIGHT - SCREEN_HEIGHT * 0.211,
        width: SCREEN_WIDTH,
        bottom: isIphoneX() ? 0 : -20,
        right: 0
    },
    headerContainer: {
        position: 'absolute',
        bottom: SCREEN_HEIGHT / 2.4,
        left: 50
    },
    button: {
        marginBottom: isIphoneX() ? SCREEN_HEIGHT * 0.1 : SCREEN_HEIGHT * 0.05,
        alignItems: 'center'
    },
    dot: {
        backgroundColor: 'white',
        height: 15,
        width: 15,
        borderRadius: 100,
        marginRight: 15
    }
})

const Screen = props => {
    return (
        <View style={Screenstyles.scrollPage}>
            <View style={Screenstyles.screen}>
                <Text style={Screenstyles.text}>{props.text}</Text>
            </View>
            {props.children}
        </View>
    )
}

const Screenstyles = StyleSheet.create({
    scrollPage: {
        width: SCREEN_WIDTH,
        flex: 1,
        justifyContent: 'center'
    },
    screen: {
        margin: 50,
        top: isIphoneX() ? SCREEN_HEIGHT * 0.05 : 0,
        position: 'absolute',
        width: '70%'
    },
    text: {
        fontSize: 24,
        fontWeight: '700',
        color: 'white',
        position: 'absolute',
        flexWrap: 'wrap',
        justifyContent: 'flex-end',
        flex: 1
    }
})

export default LandingSwipeView
