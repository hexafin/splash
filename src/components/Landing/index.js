import Landing from "./Landing";
import { connect } from "react-redux";
import { bindActionCreators } from "redux";
import {
	InviteFriends,
	getDeepLinkedSplashtag,
	SmsAuthenticate
} from "../../redux/onboarding/actions";
import { ClaimUsername } from "../../redux/user/actions";

/*
Connecting Landing to redux
*/

const mapStateToProps = state => {
	return {
		loggedIn: state.user.loggedIn,
		splashtagOnHold: state.onboarding.splashtagOnHold,
		phoneNumber: state.onboarding.phoneNumber,
		username: state.user.splashtag,
		smsAuthenticated: state.onboarding.smsAuthenticated
	};
};

const mapDispatchToProps = dispatch => {
	return bindActionCreators(
		{
			ClaimUsername,
			InviteFriends,
			getDeepLinkedSplashtag,
			SmsAuthenticate
		},
		dispatch
	);
};

export default connect(
	mapStateToProps,
	mapDispatchToProps
)(Landing);
