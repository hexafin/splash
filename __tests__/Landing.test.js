import "react-native";
import React from "react";
import Landing from "../src/components/Landing/Landing";
import LandingSwipeView from "../src/components/Landing/LandingSwipeView";

it("Landing Swipe View renders correctly", () => {
	const snap = shallow(<LandingSwipeView />);
	expect(snap).toMatchSnapshot();
});

it("Landing renders correctly", () => {
	const snap = shallow(<Landing />);
	expect(snap).toMatchSnapshot();
});
