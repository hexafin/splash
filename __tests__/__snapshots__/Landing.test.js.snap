// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Landing Swipe View renders correctly 1`] = `
<AnimatedComponent
  style={
    Array [
      Object {
        "backgroundColor": "#4547E0",
        "flex": 1,
      },
    ]
  }
>
  <Component
    style={
      Object {
        "bottom": 0,
        "flex": 1,
        "position": "absolute",
        "zIndex": 0,
      }
    }
  >
    <AnimatedComponent
      key="0-dot"
      style={
        Array [
          Object {
            "bottom": 0,
            "position": "absolute",
          },
          Object {
            "opacity": 1,
          },
        ]
      }
    >
      <Component
        resizeMode="cover"
        source={
          Object {
            "testUri": "../../../src/assets/images/screen-view1.png",
          }
        }
        style={
          Object {
            "bottom": 0,
            "height": 1052.526,
            "right": 0,
            "width": 750,
          }
        }
      />
    </AnimatedComponent>
    <AnimatedComponent
      key="1-dot"
      style={
        Array [
          Object {
            "bottom": 0,
            "position": "absolute",
          },
          Object {
            "opacity": 0,
          },
        ]
      }
    >
      <Component
        resizeMode="cover"
        source={
          Object {
            "testUri": "../../../src/assets/images/screen-view2.png",
          }
        }
        style={
          Object {
            "bottom": 0,
            "height": 1052.526,
            "right": 0,
            "width": 750,
          }
        }
      />
    </AnimatedComponent>
    <AnimatedComponent
      key="2-dot"
      style={
        Array [
          Object {
            "bottom": 0,
            "position": "absolute",
          },
          Object {
            "opacity": 0,
          },
        ]
      }
    >
      <Component
        resizeMode="cover"
        source={
          Object {
            "testUri": "../../../src/assets/images/screen-view3.png",
          }
        }
        style={
          Object {
            "bottom": 0,
            "height": 1052.526,
            "right": 0,
            "width": 750,
          }
        }
      />
    </AnimatedComponent>
  </Component>
  <Component
    style={
      Object {
        "flexDirection": "row",
        "left": 50,
        "position": "absolute",
        "top": 266.8,
        "zIndex": 2,
      }
    }
  >
    <AnimatedComponent
      key="0-dot"
      style={
        Array [
          Object {
            "backgroundColor": "white",
            "borderRadius": 100,
            "height": 15,
            "marginRight": 15,
            "width": 15,
          },
          Object {
            "opacity": 1,
          },
        ]
      }
    />
    <AnimatedComponent
      key="1-dot"
      style={
        Array [
          Object {
            "backgroundColor": "white",
            "borderRadius": 100,
            "height": 15,
            "marginRight": 15,
            "width": 15,
          },
          Object {
            "opacity": 0.25,
          },
        ]
      }
    />
    <AnimatedComponent
      key="2-dot"
      style={
        Array [
          Object {
            "backgroundColor": "white",
            "borderRadius": 100,
            "height": 15,
            "marginRight": 15,
            "width": 15,
          },
          Object {
            "opacity": 0.25,
          },
        ]
      }
    />
  </Component>
  <AnimatedComponent
    horizontal={true}
    onScroll={[Function]}
    pagingEnabled={true}
    scrollEventThrottle={16}
    showsHorizontalScrollIndicator={false}
  >
    <Screen
      index={0}
      text="Splash is your cryptocurrency wallet."
    />
    <Screen
      index={1}
      text="Send money to anyone.
Buy anything."
    />
    <Screen
      index={2}
      text="Bye, bye big banks.
Hello Splash."
    />
  </AnimatedComponent>
  <Component
    style={
      Object {
        "alignItems": "center",
        "marginBottom": 100.05,
      }
    }
  >
    <Button
      large={true}
      onPress={[Function]}
      round={true}
      title="Get your wallet"
    />
    <TouchableOpacity
      activeOpacity={0.2}
      onPress={[Function]}
    >
      <Component
        style={
          Object {
            "color": "white",
            "fontSize": 16,
            "fontWeight": "700",
            "paddingTop": 20,
          }
        }
      >
        Or, recover your Splash wallet
      </Component>
    </TouchableOpacity>
  </Component>
</AnimatedComponent>
`;

exports[`Landing renders correctly 1`] = `
<Component
  style={
    Object {
      "backgroundColor": "#FFFFFF",
      "flex": 1,
      "justifyContent": "space-between",
      "paddingBottom": 20,
      "paddingTop": 20,
      "position": "relative",
    }
  }
>
  <AnimatedWaves />
  <Component
    style={
      Object {
        "flex": 1,
        "flexDirection": "column",
        "padding": 30,
      }
    }
  >
    <Component
      style={
        Object {
          "alignItems": "center",
          "flexDirection": "row",
          "justifyContent": "center",
          "paddingTop": 63,
        }
      }
    >
      <Component
        source={
          Object {
            "testUri": "../../../src/assets/images/splash-logo.png",
          }
        }
        style={
          Object {
            "height": 30,
            "margin": 5,
            "width": 22.5,
          }
        }
      />
      <Component
        style={
          Object {
            "color": "#333333",
            "fontSize": 22,
            "fontWeight": "400",
            "paddingBottom": 4,
          }
        }
      >
        Splash
      </Component>
    </Component>
    <Component
      style={
        Object {
          "alignItems": "center",
          "flexDirection": "column",
          "marginTop": 30,
          "padding": 0,
        }
      }
    >
      <Component
        style={
          Object {
            "color": "#333333",
            "fontSize": 32,
            "fontWeight": "500",
            "textAlign": "center",
          }
        }
      >
        Splash is your wallet
      </Component>
      <Component
        style={
          Object {
            "color": "#ADADAD",
            "fontSize": 22,
            "marginTop": 20,
            "textAlign": "center",
          }
        }
      >
        Make it personal with a splashtag
      </Component>
    </Component>
  </Component>
  <Component
    style={
      Object {
        "marginBottom": 10,
        "padding": 20,
      }
    }
  >
    <Button
      onPress={[Function]}
      primary={false}
      title="Get started"
    />
  </Component>
</Component>
`;
